src/App.js

```diff
@@ -25,7 +25,10 @@ function App() {
           element={<Categories setCategoryName={setCategoryName} />}
         />
         <Route path="/reviews/:category" element={<Reviews />} />
-        <Route path="/reviews/article/:review_id" element={<SingleReview />} />
+        <Route
+          path="/reviews/article/:review_id"
```

Comment:
Just a small thing, but I'd probably expect to find a review on "/review/:review_id" instead of "/reviews/article/:review_id"